### DISCUSSION TITLE
Output

### DISCUSSION THREAD

**Discussion by Krisztian0801** [apache/netbeans#4810] [Q&A]
At the output lines when I print a string with hungarian charekters that charekter shows as a squar. I set the encoding UTF-8 but it is not works . What can I do?
![Képernyőkép 2022-10-18 112418](https://user-images.githubusercontent.com/91634567/196391628-d904a5f9-d495-4ca6-873a-16d116fb1fcb.png)

**Comment by neilcsmith-net** [comment]
> I set the encoding UTF-8 but it is not works . What can I do?

_How_ did you set the encoding?  Are you on Windows?

Please read #4771 and other issues linked from there.
